tests/e2e/deletion_test.go

```diff
@@ -68,7 +70,7 @@ func (tc *testContext) testApplicationDeletion(component components.ComponentInt
 
 	if err := wait.PollUntilContextTimeout(tc.ctx, tc.resourceRetryInterval, tc.resourceCreationTimeout, false, func(ctx context.Context) (bool, error) {
 		appList, err := tc.kubeClient.AppsV1().Deployments(tc.applicationsNamespace).List(ctx, metav1.ListOptions{
-			LabelSelector: odhLabelPrefix + component.GetComponentName(),
+			LabelSelector: labels.ODH.Component(component.GetComponentName()),
 		})
 		if err != nil {
 			log.Printf("error listing component deployments :%v. Trying again...", err)
```

Comment:
there was a hunk here in the original patch

Reply:
So, what about changing it to `log.Printf("error getting component deployments :%v. Trying again...", err)`  like in the original patch?

Reply:
i did the fix on this in the latest commit